hilti/src/rt/backtrace.cc

```diff
@@ -1,38 +1,61 @@
 // Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
 
-#include "rt/backtrace.h"
-
 #include <cstring>
 
 #include <hilti/rt/autogen/config.h>
+#include <hilti/rt/backtrace.h>
```

Comment:
Let's continue to include the matching header first, https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes. We are not fully consistent on this, but I tried to fix this in files I touched.

Reply:
accidental change, will fix